pkg/provider-local/controller/ingress/add.go

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) 2022 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
```

Comment:
Please add this controller with to https://github.com/gardener/gardener/blob/2a5dacb7f89414d7ed28a98fd9688283fb3977cc/charts/gardener/provider-local/extension/values.yaml#L20